src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -2584,9 +2584,8 @@ def test_webapp_local_context(self, resource_group):
 
 
 class WebappOneDeployScenarioTest(ScenarioTest):
-    @live_only()
     @ResourceGroupPreparer(name_prefix='cli_test_webapp_OneDeploy', location=WINDOWS_ASP_LOCATION_WEBAPP)
-    def test_one_deploy(self, resource_group):
+    def test_one_deploy_scm(self, resource_group):
         webapp_name = self.create_random_name('webapp-oneDeploy-test', 40)
         plan_name = self.create_random_name('webapp-oneDeploy-plan', 40)
         war_file = os.path.join(TEST_DIR, 'sample.war')
```

Comment:
Since the `@live_only()` label of this test has been removed, our CI pipeline now run it's recording test.

> war_file = os.path.join(TEST_DIR, 'sample.war')

Please move the `sample.war` file to the `src/azure-cli/azure/cli/command_modules/appservice/tests/latest/data` folder, otherwise it will cause the execution of package tests in our Batch CI pipeline to fail.